ui/app/AppLayouts/Onboarding2/controls/LoginUserSelector.qml

```diff
@@ -0,0 +1,160 @@
+                userSelectorButton.image = item.thumbnailImage
+                userSelectorButton.colorHash = item.colorHash
+                userSelectorButton.colorId = item.colorId
+                userSelectorButton.keycardCreatedAccount = item.keycardCreatedAccount
+            }
+        }
+    }
+
+    function close() {
+        dropdown.close()
+    }
+
+    QtObject {
+        id: d
+
+        readonly property int maxPopupHeight: 300
+        readonly property int delegateHeight: 64
+    }
+
+    contentItem: LoginUserSelectorDelegate {
+        id: userSelectorButton
+        background: Rectangle {
+            color: userSelectorButton.hovered ? Theme.palette.baseColor2 : "transparent"
+            border.width: 1
+            border.color: Theme.palette.baseColor2
+            radius: Theme.radius
+        }
+        rightPadding: spacing + Theme.padding + chevronIcon.width
+        StatusIcon {
+            id: chevronIcon
+            anchors.right: parent.right
+            anchors.rightMargin: Theme.padding
+            anchors.verticalCenter: parent.verticalCenter
+            icon: "chevron-down"
+            color: Theme.palette.baseColor1
+        }
+        keycardLocked: root.currentKeycardLocked
+        onClicked: dropdown.opened ? dropdown.close() : dropdown.open()
+    }
+
+    StatusDropdown {
+        id: dropdown
+
+        y: parent.height + 4
+        width: root.width
+
+        verticalPadding: Theme.halfPadding
+        horizontalPadding: 0
+
+        contentItem: ColumnLayout {
+            spacing: 0
+            StatusListView {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                Layout.maximumHeight: d.maxPopupHeight
+                id: userSelectorPanel
+                model: SortFilterProxyModel {
+                    id: proxyModel
+                    sourceModel: root.loginAccountsModel
+                    sorters: StringSorter {
+                        roleName: "order"
+                        sortOrder: Qt.AscendingOrder
+                    }
+                    filters: [
+                        ValueFilter { // don't show the currently selected item
+                            roleName: "keyUid"
+                            value: userSelectorButton.keyUid
+                            inverted: true
+                        }
+                    ]
+                }
+                implicitHeight: contentHeight
+                spacing: 0
+                delegate: LoginUserSelectorDelegate {
+                    width: ListView.view.width
+                    height: d.delegateHeight
+                    keyUid: model.keyUid
+                    label: model.username
+                    image: model.thumbnailImage
+                    colorId: model.colorId
+                    colorHash: model.colorHash
+                    // keycardLocked // FIXME needed here too? should be coming from the model imo
```

Comment:
What happens if this is set to true? Does it mean that it uses a keycard or that the keycard is locked?

Reply:
The latter, we can arrive there are too many failed PIN attempts

Reply:
But I verified with Ben that it's not needed in the combo (that keycard may not be inserted yet); will remove